pycbc/psd/read.py

```diff
@@ -60,12 +60,19 @@ def from_numpy_arrays(freq_data, noise_data, length, delta_f, low_freq_cutoff):
     freq_data = freq_data[data_start:]
     noise_data = noise_data[data_start:]
 
+    if (length - 1) * delta_f > freq_data[-1]:
+        print('Requested number of samples exceeds the highest '
```

Comment:
Not sure if this should be a print statement. How is this usually handled in PyCBC @ahnitz?

Reply:
If this is a failure condition that should result in the termination of the program, raise a ValueError with this message. Otherwise, if it is just to provide information to the user, call `logging.info()`, `logging.warn()` or `logging.error()`.